assets/css/styles.css

```diff
@@ -227,6 +227,18 @@
   margin-top: 0;
```

Comment:
- [ Optional ] Please the position and the size of the background image should match the design 
Your design 
![image](https://user-images.githubusercontent.com/74744787/229237163-27afae8a-c697-4422-ad42-6867becf8972.png)
Figma 
![image](https://user-images.githubusercontent.com/74744787/229237201-de73ff0a-d6c5-4941-9c6d-710a1f24a390.png)

Reply:
Thanks for your suggestions, dear @MeqdamAlqudah ....

Maybe because I'm using a max resolution of 1280px on my desktop computer, I did some tests in the Figma resolution 1440x916 (laptop) and I never lose the responsiveness.

**1440x916px Horizontal**

![1440x916-Horizontal](https://user-images.githubusercontent.com/86327867/229302085-a24f91f5-01a5-4a73-9938-28fd1c7c21ad.jpg)

**1440x916px Vertical**

![1440x916-Vertical](https://user-images.githubusercontent.com/86327867/229302278-fcea9120-6b32-4aa0-8484-8ab43e8df62c.jpg)

I guess that it's working all good, my brother (I did some changes in the cards section where the cards change their content (text content) working for more than 3 hours yesterday) trying to do my job as near as possible to the figma tempate. 

Thanks for your valuable feedback, dear friend.

Javier.-

Reply:
@jlcarrascof 
This problem happens because you're using the mobile background image for desktop version. On desktop version the background image should change.
**We can see on this screenshot that the mobile version background image is used even on screen size > 780px**
![image](https://user-images.githubusercontent.com/74411135/229304448-739cd26b-746d-4524-b317-665d2a9f9878.png)